readyapi4j-groovy-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/pro/ProDslDelegate.groovy

```diff
@@ -0,0 +1,29 @@
+package com.smartbear.readyapi4j.dsl.pro
+
+import com.smartbear.readyapi4j.dsl.DslDelegate
+
+/**
+ * The delegate responding to commands inside the "recipe" closure of ServerTestDsl.
+ */
+class ProDslDelegate extends DslDelegate {
```

Comment:
Me and Manne had a chat about it and we called it ProDslDelegate because we might want to add support for local execution of Pro steps in future, using SoapUI Pro or Ready! API dependencies.